src/lib/colors/index.ts

```
@@ -62,3 +62,12 @@ export const element_color_schemes = {
 } as const
 
 export const default_element_colors = { ...vesta_hex }
+
+// Helper function to detect if a value is a color string
+export const is_color = (val: unknown): val is string => {
+  if (typeof val !== `string`) return false
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)